_posts/2020-07-13-i-know-html-now-what.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: I know html, now what?
+date: 2020-07-13 16:56 +0100
+learn:
+  - hosting a basic website on vercel, netlify, and github pages
+---
+
+# I know HTML, now what?
+
+[Skip to the mission](#the-mission)
+
+## Background
+
+After completeling that [one codeacadmey
+course](https://www.codecademy.com/learn/learn-html), you know how tags work,
+some html elements, maybe even some [a11y](https://a11yproject.com/) if you're
+lucky. You can probably make a simple web page.
+
+Now, you want to show off your progress... How exactly do you do that? How do
+your favourite websites actually, website?
+
+Well, you might have heard that the internet is made up of loads of computers
+talking to eachother, and this essentialy true. A website is just a computer
```

Comment:
this *is*

Reply:
?

Reply:
I believe the sentence at line 24 should be fixed to: "this *is* essentially true".